### DISCUSSION TITLE
preview or open file?

### DISCUSSION THREAD

**Discussion by reagle** [Byron/dua-cli#152] [Q&A] [answered]
Is there a way to preview or open a file (e..g, it's a simple text file).

**Comment by Byron** [comment, marked as answer]
Does shift + o work? If not, shift + ? should bring up a help menu that has the shortcut you seek.

**Comment by reagle** [comment]
I looked at help, but it appears to be off screen. How do I scroll the help message?

<img width="474" alt="SCR-20230314-gssn" src="https://user-images.githubusercontent.com/596814/224980877-43e96ecc-29b6-46be-b04d-72dc7c89eb37.png">

**Reply by Byron** [reply]
<img width="728" alt="Screenshot 2023-03-14 at 18 40 29" src="https://user-images.githubusercontent.com/63622/225091912-3aeb22dd-83d0-4d1b-918b-a2cbfd76544b.png">

It's interesting that the inline help (red box) is entirely missing for you. Vim controls scroll up and down, `j`, `k` and others do the trick.

**Reply by reagle** [reply]
Yes, when the terminal window is relatively small, most of the stuff in red disappears. I didn't realize there was more help, that the help window could be scrolled, and for some reason it was jittery at first, but it seems to work if I'm patient. Perhaps if there was something at the bottom that indicated there's more below I wouldn't have made this mistake. In any case, thanks for your help.

**Comment by Byron** [comment]
> I didn't realize there was more help, that the help window could be scrolled [..]

Yes, there is no actual windowing system to know what's on and off a virtual screen, everything has to be done 'by hand'.

> and for some reason it was jittery at first [..]

Probably it was still scanning.

>  In any case, thanks for your help.

You are welcome.
